ehrapy/api/preprocessing/encoding/_encode.py

```diff
@@ -228,13 +228,12 @@ def _encode(
         var_to_encoding = {} if "var_to_encoding" not in adata.uns.keys() else adata.uns["var_to_encoding"]
         encoded_x = None
         encoded_var_names = adata.var_names.to_list()
-
         with Progress(
             "[progress.description]{task.description}", BarColumn(), "[progress.percentage]{task.percentage:>3.0f}%"
         ) as progress:
 
             for encoding_mode in encodings.keys():
-                task = progress.add_task(f"[red] Setting up {encoding_mode}", total=len(encodings[encoding_mode]))
+                task = progress.add_task(f"[red] Setting up {encoding_mode}", total=1)
```

Comment:
Is it possible to just remove the total here?
Isn't this like total=1?

Reply:
From rich docs: `add_task(description: str, start: bool = True, total: float = 100.0, completed: int = 0, visible: bool = True, **fields: Any) → TaskID`.

Defaults to `100.0`